lib/fs.js

```diff
@@ -2000,7 +2000,8 @@ SyncWriteStream.prototype.destroy = function() {
 
 SyncWriteStream.prototype.destroySoon = SyncWriteStream.prototype.destroy;
 
-fs.mkdtemp = function(prefix, options, callback) {
+fs.mkdtemp = function(prefix, options, callback_) {
+  let callback = maybeCallback(callback_);
```

Comment:
minor nit: might change this to `var callback` to be consistent with the rest of `fs`. Also `let` in general is still slow IIRC, although whether that will matter in the grand scheme of things in this particular case is hard to say.